include/interfaces/turbulence_k_omega.h

```diff
@@ -52,8 +53,47 @@ class KOmega
   void
   set_matrix_couplings(std::vector<std::string> &couplings) const;
 
+  virtual void connect_to_signals() const
+  {
+    auto &signals = this->get_signals();
+    auto &pcout = this->get_pcout();
+    signals.begin_make_grid_fe.connect(
+      [&]()
+    {
+      pcout << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl
+            << " --------------------- Eikonal Equation -----------------------"
+            << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl << std::endl;
+      distance.run ();
+      pcout << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl
+            << " --------------------- Eikonal Equation -----------------------"
+            << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl << std::endl;
+      const typename LAC::VectorType &distance_sol = distance.get_solution();
+      const DoFHandler<dim,spacedim> &distance_dh  = distance.get_dof_handler();
+      Functions::FEFieldFunction<dim, DoFHandler<dim,spacedim>, typename LAC::VectorType> dist(distance_dh, distance_sol);
+      {
+        Point<spacedim> p;
+        p[0] = 0.5;
+        p[1] = 0.5;
+        pcout << "-->" << dist.value(p,0) << std::endl;
```

Comment:
@asartori86 e @luca-heltai. Sono ancora in fase test. Tuttavia sembra che l'accoppiamento riesca a risolvere l'equazione eikonale senza troppi problemi.
Solo che nel momento in cui definisco la FEFieldFunction se ho bisogno di interrogarla, diciamo con un pcout, il codice va in segfault. Idee? Se commento questa riga tutto ok.